### ISSUE TITLE
list index out of range

### ISSUE THREAD

**Issue by yangdaowu** [eric88525/Speaker-Separation#1]
Hello, I have a problem when I use your code training:
![image](https://user-images.githubusercontent.com/64669901/124898641-f71ef880-e011-11eb-9352-440e441b0f33.png)
Is it because of the number of speakers, a few speakers?

**Comment by eric88525** [comment]
Hello yangdaowu.

I think the problem is caused by opt error. Plz check your opt look like this.

There are two scp file path in dataroot_targets , and each scp files record the train/val wav files path.

![config](https://user-images.githubusercontent.com/39957434/124904374-87137100-e017-11eb-8369-07065c07db1b.png)

**Comment by yangdaowu** [comment]
Yes, I have solved this problem.But I have encountered memory, in addition to modifying BATCH_SIZE, what you need to modify what can reduce memory.

**Comment by eric88525** [comment]
Well.. changing the model parameter maybe is not a good way because it effects performance.
 I suggest that you use shorter wav file to train model or set batch_size=1 or 2.
You can also run program on google colab , it provides great GPU like T4(16GB) , P-100(16GB).
